loops-arrays-objects.js

```diff
@@ -22,48 +26,87 @@ const scores = [
 
 // ==========================================
 // Opdracht 1b
-// Breid je script uit door voor iedere student de score om te rekenen naar een letter en dit in de terminal te printen
-// < 60 = F, < 70 = D, <80 = C, <90 B, <100 = A
-// Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
-// Verwachtte uitkomsten:
+// Expand your script by converting the score for each student into a letter and printing this in the terminal
+// // < 60 = F, < 70 = D, <80 = C, <90 B, <100 = A
+// // Make sure this would also work if the array contained 100 or 200 numbers!
+// // Expected outcomes:
 // B
 // C
 // A
 // D
 // ==========================================
 
+for(let i=0; i<scores.length; i++){
+    const studentScore =scores[i].score;
+    if(studentScore >=90){
+        scores[i].grade = 'A';
+    }else if(studentScore<90 && studentScore >=80){
+        scores[i].grade='B';
+    }else if(studentScore <80 && studentScore >=70){
+        scores[i].grade='C';
+    }else if(studentScore<70 && studentScore>=60) {
+        scores[i].grade = 'D'
+    }else {
+        scores[i].grade = 'F';
+    }
```

Comment:
If you would turn around the if-statements some of the statements should be implicitly result true
 ```suggestion
    if(studentScore < 60){
        scores[i].grade = 'F';
    }else if(studentScore < 70){
        scores[i].grade='D';
    }else if(studentScore < 80){
        scores[i].grade='C';
    }else if(studentScore < 90) {
        scores[i].grade = 'B'
    }else {
        scores[i].grade = 'A';
    }
```